### ISSUE TITLE
Color customization

### ISSUE THREAD

**Issue by Ewocker** [cloudnativelabs/kube-shell#56]
how can we customize the color?

**Comment by rsrini7** [comment]
Yes i am also looking for the same.. how to change?  Not able to read documentation. But excellent tool. 

![2018-09-18 22_11_15-kube-shell](https://user-images.githubusercontent.com/5178105/45702588-db5fb480-bb8f-11e8-8654-a8fff14e8325.png)

**Comment by Vanosz** [comment]
Have the same issue. And can't find any solutions. These colors are too acidly and I can't read correctly any of drop-down hints.
![image](https://user-images.githubusercontent.com/41544334/94832209-4a767180-0416-11eb-85f8-b9c91ed2f9d7.png)
